angular_token_auth.js

```diff
@@ -13,6 +13,8 @@
         AUTH_HEADER_PREFIX: 'Token',
         ALLOWED_HOSTS: [],
         COOKIE_PATH: null
+        // Optional settings examples:
+        // STORAGE_METHOD: 'localstorage',
```

Comment:
You mean add comments for the existing options? Definitely a good idea, but should probably be another issue/PR